terraform/eks-cloudwatch/variables.tf

```diff
@@ -0,0 +1,22 @@
+# ------------------------------------------------------------------------
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+# -------------------------------------------------------------------------
+
+variable "eks_cluster_name" {
+  default = "aws-otel-testing-framework-eks"
+}
+
+variable "sample_app_image_repo" {
+  default = "611364707713.dkr.ecr.us-west-2.amazonaws.com/otel-test/container-insight-samples"
```

Comment:
The source code can be found following our official setup guidance, it is maintained in another AWS repo: https://github.com/aws/aws-app-mesh-examples/tree/master/walkthroughs/howto-k8s-http-headers.
Ref:
https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/ContainerInsights-Prometheus-Sample-Workloads-appmesh-EKS.html

I didn't put it under the sample-apps folder because I personally think:
1. It increases the effort to sync this copy with the original one.
2. I can't see any potential user that is interested in running the integration tests locally. If this image would only be used by aoc integration test, I don't see a strong reason to maintain it. What's the benefit?
3. We don't aim to test sample apps, they serve for testing aoc. A stable version of sample app is enough to meet the requirement.

Reply:
1. It increases the effort to sync this copy with the original one. make sense.
2. I can't see any potential user that is interested in running the integration tests locally. If this image would only be used by aoc integration test, I don't see a strong reason to maintain it. What's the benefit? -> who is maintaining this sample app now? does it have workflow for auto-build? how do we track the version?
3. We don't aim to test sample apps, they serve for testing aoc. A stable version of sample app is enough to meet the requirement. ->  would we need performance test?

Reply:
i wonder if this sample app is just for aoc integration-test, why don't we move it into this repo? i'm okay with that if this sample app has multiple usages than aoc inregration-test

Reply:
> who is maintaining this sample app now? does it have workflow for auto-build? 
I believe it's EKS team, checking the contributors. 
Per [README](https://github.com/aws/aws-app-mesh-examples/blob/master/walkthroughs/howto-k8s-http-headers/README.md), they've provided a deploy script that will build the image, and deploy the relating resources to the cluster (cleansing should still be taken cared of by users).
> how do we track the version?
Images are for sample apps, they don't need version tracking, or so I thought.
> would we need performance test?
I'd like to see it happen. But we don't have concrete plan yet. Yet that's unrelated to sample app images IIUC.
> this sample app is just for aoc integration-test, why don't we move it into this repo? 
Apparently it's not for aoc integration-test, but is recommended by CloudWatch for AppMesh monitoring quick start.

Reply:
pls help to link the source code in the comment.

Reply:
Added def and source code links to the variable.